XamlControlsGallery/ItemPage.xaml

```diff
@@ -94,6 +96,8 @@
                         <Setter Target="seeAlsoPanel.(Grid.Column)" Value="0" />
                         <Setter Target="seeAlsoPanel.(Grid.ColumnSpan)" Value="3" />
 
+                        <Setter Target="SourcePanel.(Grid.ColumnSpan)" Value="1" />
```

Comment:
Is this not the default? Why does this need to be set?

Reply:
We define the SourcePanel with `Grid.ColumnSpan=3`. In narrow layouts we don't want it to stretch over other content, so we set ColumnSpan=1 in that case.